veolia-idf-domoticz.py

```diff
@@ -238,159 +286,250 @@ def init_browser_firefox(self):
+        self.print(
+            "Start the browser", end=""
+        )  #############################################################
         try:
-            self.__browser = webdriver.Chrome(executable_path=self.configuration["chromedriver"],chrome_options=options)
+            self.__browser = webdriver.Chrome(
+                executable_path=self.configuration["chromedriver"],
+                chrome_options=options,
+            )
             self.__browser.maximize_window()
-            self.__wait = WebDriverWait(self.__browser, int(self.configuration["timeout"]))
+            self.__wait = WebDriverWait(
+                self.__browser, int(self.configuration["timeout"])
+            )
         except Exception:
             raise
         else:
-            self.print(st = "OK")
-        pass
+            self.print(st="OK")
 
-    def sanity_check(self):
+    def sanity_check(self, debug=False):  # pylint: disable=unused-argument
 
-        self.print("Check download location integrity", end="") #############################################################
+        self.print(
+            "Check download location integrity", end=""
+        )  #############################################################
         if os.path.exists(self.__full_path_download_file):
-            self.print(self.__full_path_download_file + " already exists, will be removed", "WW")
+            self.print(
+                self.__full_path_download_file
+                + " already exists, will be removed",
+                "WW",
+            )
         else:
             try:
-                f = open(self.__full_path_download_file, "a+").close()
+                open(self.__full_path_download_file, "a+").close()
             except Exception as e:
-                raise RuntimeError('"' + self.__full_path_download_file + '" ' + e.strerror)
+                raise RuntimeError(
+                    '"%s" %s' % (self.__full_path_download_file, e,)
+                )
             else:
                 self.print(st="ok")
 
-        self.print("Remove temporary download file", end="") #############################################################
+        #############################################################
         try:
+            self.print( "Remove temporary download file", end="")
             os.remove(self.__full_path_download_file)
         except Exception:
             raise
         else:
             self.print(st="ok")
 
-        self.print('Check if "geckodriver" is intalled properly', end="") #############################################################
-        if os.access(self.configuration['geckodriver'], os.X_OK):
-            self.print(st="ok")
-        else:
-            raise OSError('"' + self.configuration['geckodriver'] + '" is not executable or not found')
+        self.print(
+            'Check if "geckodriver"+"firefox" or "chromedriver"+"chrome" is installed properly', end=""
```

Comment:
changer 
            'Check if "geckodriver"+"firefox" or "chromedriver"+"chrome" is installed properly', end=""
en
            'Check if "geckodriver"+"firefox" or "chromedriver"+"chrome" are installed ', end=""

Reply:
Perso je penche plutôt pour le 'is' :  
L'une ou l'autre combinaison est-elle installée?

Il faudrait demander à un anglophone pour en avoir le coeur net, mais perso je maintien le "is".  En tous cas, la proposition ne "sonne" pas bien, il faudrait la reformuler.  (J'ai eu la télé en VO depuis ma naissance...).

Reply:
Oui je vais demandé.  Je vois "geckodriver"+"firefox"  comme 2 composants donc "are" :) 
sinon j'ai enlevé le "properly" pour que la ligne tienne dans 75 characters

Reply:
Une solutions dans 31e69f1.

Reply:
🤟🏻 
tout est bon pour toi, je merge ?

Reply:
Pour moi c'est bon.